docs/getting_started_users/index.rst

```diff
@@ -0,0 +1,48 @@
+
+.. _getting_started_users:
+
+*************************
+Getting Started For Users
+*************************
+
+.. warning::
+
+   The following guide is for *users*. As of now this will just cover
+   the basic installation procedure. This guide will be extended in
+   the future.
+
+
+------------------------
+Get the ctapipe software
+------------------------
+
++++++++++++++++++++++++++++++
+How to get the latest version
++++++++++++++++++++++++++++++
+
+We recommend using the ``mamba`` package manager, which is a C++ reimplementation of ``conda``.
+It can be found `here <https://github.com/mamba-org/mamba>`_.
+Once you created a new virtual environment, you can install ``ctapipe`` with the following command::
+
+  mamba install -c conda-forge ctapipe
+
+or with pip::
+
+  pip install ctapipe
+
+
++++++++++++++++++++++++++++++
+How to get a specific version
++++++++++++++++++++++++++++++
+
+To install a specific version of ``ctapipe`` you can use the following commands::
+
+  CTAPIPE_VER=0.17.0
+  wget https://raw.githubusercontent.com/cta-observatory/ctapipe/v$CTAPIPE_VER/environment.yml
+  mamba env create -n cta -f environment.yml
+  mamba activate cta
+  mamba install -c conda-forge ctapipe=$CTAPIPE_VER
```

Comment:
This is not really installing a specific version. That would be
```
pip install ctapipe==0.17.0  # or
mamba install ctapipe=0.17.0
```

This is setting up the development environment (although with a different name).

I think this part should move to getting started for devs (excluding changing the name).